usage-container/README.md

```diff
@@ -0,0 +1,122 @@
+# Container workflow for HADDOCK3
+
+This repository provides everything you need to build, run, and extend an **Apptainer/Singularity** container for HADDOCK3. [**HADDOCK3**](https://doi.org/10.1021/acs.jcim.5c00969)(High Ambiguity Driven protein–protein Docking) is a flexible, information-driven software suite for modeling biomolecular complexes using experimental and theoretical restraints. **Docker**, **Singularity**, and **Apptainer** are containerization platforms that package applications and all their dependencies into lightweight, portable images, ensuring reproducible execution across different environments.
+
+A ready-to-use Docker image for HADDOCK3 is published on the GitHub container registry, simply pull:
+```bash
+docker pull ghcr.io/haddocking/haddock3:2025.5.0
```

Comment:
We have now released 2025.8.0  -  so should this command be made more generic to pull the latest version?

And add a second command to pull a specific version